src/styles/main.scss

```diff
@@ -1 +1,76 @@
-// put code here
+body {
+  box-sizing: border-box;
+  margin: 0;
+}
+
+.wrapper {
+  height: 100vh;
+  display: flex;
+  justify-content: center;
+  align-items: center;
+}
+
+.stopwatch {
+  width: 80vmin;
+  height: 80vmin;
+  border: 1vmin dotted #000;
+  border-radius: 50%;
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  position: relative;
+
+  &__hand {
+    transform-origin: bottom center;
+    position: absolute;
+  }
+
+  &__hand--minute {
+    width: 3vmin;
+    height: 20vmin;
+    background-color: #0700ff;
+    animation: minute 3600s infinite steps(60);
+  }
+
+  &__hand--second {
+    width: 1.5vmin;
+    height: 38vmin;
+    background-color: #2c8000;
+    animation: second 60s infinite linear;
+  }
+
+  &__center {
+    position: absolute;
+    display: flex;
+    width: 5vmin;
+    height: 5vmin;
+    background-color: #f6a603;
+    border-radius: 50%;
+  }
+}
+
+@keyframes minute {
+  0% {
+    transform: translate(0, -50%) rotate(0deg);
+  }
+  100% {
+    transform: translate(0, -50%) rotate(360deg);
+  }
+}
+
+@keyframes second {
+  0% {
+    transform: translate(0, -50%) rotate(0deg);
+  }
+  100% {
+    transform: translate(0, -50%) rotate(360deg);
+  }
+}
+
+.stopwatch--speed-up .stopwatch__hand--second {
+  animation-duration: 10s;
```

Comment:
write it in this selector (with all other properties):
`&__hand--second`

`.stopwatch--speed-up` class seems redundant - remove it

Reply:
![Screenshot_1](https://github.com/mate-academy/layout_stop-watch/assets/149514552/6ffce66d-3d7c-41f0-888c-9e33d8ba8336)

Reply:
sorry, now I see `stopwatch--speed-up` was needed for tests (it was recently updated) - my bad
Anyway - everything looks good, task is already approved :heavy_check_mark: